pom.xml

```diff
@@ -126,6 +126,12 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>org.glassfish.hk2</groupId>
+                <artifactId>hk2-utils</artifactId>
+                <version>2.4.0-b32</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
Tracked it down.

**origin/master**
```
[INFO] com.yahoo.elide:elide-integration-tests:jar:1.0.0.4-SNAPSHOT
[INFO] +- com.yahoo.elide:elide-core:jar:1.0.0.4-SNAPSHOT:compile
[INFO] |  +- com.google.guava:guava:jar:16.0.1:compile
[INFO] |  +- org.glassfish.hk2:hk2-api:jar:2.4.0-b25:compile
[INFO] |  |  +- org.glassfish.hk2:hk2-utils:jar:2.4.0-b25:compile
[INFO] |  |  \- org.glassfish.hk2.external:aopalliance-repackaged:jar:2.4.0-b25:compile
[INFO] |  +- org.antlr:antlr4-runtime:jar:4.5:compile
```

**updateDeps** branch
```
[INFO] com.yahoo.elide:elide-integration-tests:jar:1.0.0.4-SNAPSHOT
[INFO] +- com.yahoo.elide:elide-core:jar:1.0.0.4-SNAPSHOT:compile
[INFO] |  +- com.google.guava:guava:jar:18.0:compile
[INFO] |  +- org.glassfish.hk2:hk2-api:jar:2.4.0-b25:compile
[INFO] |  |  \- org.glassfish.hk2.external:aopalliance-repackaged:jar:2.4.0-b25:compile
[INFO] |  +- org.antlr:antlr4-runtime:jar:4.5.1-1:compile
```

Not entirely sure what changed here. In any case, it appears to be fixed by bumping the hk2 version (see updated commit.)